### PR TITLE
1.8.0 release notes and highlights

### PR DESCRIPTION
**Pull request by sebgl**
Add release notes and release highlights for the upcoming ECK 1.8.0.

### CODE REVIEW

File: docs/release-notes/1.8.0.asciidoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+[[release-notes-1.8.0]]
+== {n} version 1.8.0
+
+[[breaking-1.8.0]]
+[float]
+=== Breaking changes
+
+* Remove non-functional pre-stop hook parts {pull}4801[#4801] (issue: {issue}4698[#4698])
+
+The operator does not rely on the `$PRE_STOP_MAX_WAIT_SECONDS` environment variable to control <<{p}-prestop,Elasticsearch Pods PreStop hook>> anymore. Use `$PRE_STOP_ADDITIONAL_WAIT_SECONDS` instead (defaults to 50 seconds).
```
Comment (sebgl) [inline docs/release-notes/1.8.0.asciidoc:13]: Manually added some additional notes on the breaking change here.

File: docs/release-notes/1.8.0.asciidoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+* Note the next minor ECK version (1.9.0) will drop support for Openshift 3.11.
```
Comment (sebgl) [inline docs/release-notes/1.8.0.asciidoc:15]: Added that as breaking change to make sure people notice, but it's also explained in the release highlights. So maybe I should remove it from here?

File: docs/release-notes/1.8.0.asciidoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+[float]
+=== Misc
+
+* Update module gopkg.in/yaml.v1 to v2 {pull}4788[#4788]
+* Update module github.com/fsnotify/fsnotify to v1.5.0 {pull}4778[#4778]
+* Update golang Docker tag to v1.17.0 {pull}4766[#4766]
+* Update module sigs.k8s.io/controller-runtime to v0.9.6 {pull}4747[#4747]
+* Update module go.uber.org/zap to v1.19.0 {pull}4742[#4742]
+* Update golang Docker tag to v1.16.7 {pull}4741[#4741]
+* Update module sigs.k8s.io/controller-tools to v0.6.2 {pull}4697[#4697]
```
Comment (sebgl) [inline docs/release-notes/1.8.0.asciidoc:42]: I removed the controller-runtime 0.10.0 related PRs, since reverted.

Comment (thbkrkr) [reply docs/release-notes/1.8.0.asciidoc:42]: Update golang Docker tag to v1.17.0 #4766
Update golang Docker tag to v1.16.7 #4741
:shrug:

File: docs/release-notes/highlights-1.8.0.asciidoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+[id="{p}-180-helm-chart-ga"]
+==== ECK Helm Chart graduating from beta to GA
+
+The ECK operator Helm chart is now generally available. It provides an easy way to install and configure the operator along with its Custom Resource Definitions.
+
+[float]
+[id="{p}-180-openshift-311"]
+==== Last minor version to support Openshift 3.11
+
+The ECK 1.8.x minor release series is the last to support Openshift 3.11. Starting ECK 1.9.0, Openshift 3.11 won't be supported anymore.
```
Comment (david-kow) [inline docs/release-notes/highlights-1.8.0.asciidoc:20]: Should we link to the support policy here?

File: docs/release-notes/1.8.0.asciidoc (hunk 2 above)
Comment (david-kow) [reply docs/release-notes/1.8.0.asciidoc:15]: I think we should leave it here, I'd rather duplicate that info than risk someone getting surprised.

**Review by david-kow — APPROVED** [review]
LGTM

File: docs/release-notes/1.8.0.asciidoc (hunk 3 above)
Comment (sebgl) [reply docs/release-notes/1.8.0.asciidoc:42]: Let's keep the most recent one

File: docs/release-notes/highlights-1.8.0.asciidoc (hunk 4 above)
Comment (sebgl) [reply docs/release-notes/highlights-1.8.0.asciidoc:20]: Good idea!

File: docs/release-notes/1.8.0.asciidoc [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+* Drop "properties: {}" from empty-v1alpha1-patch {pull}4813[#4813]
+* Remove beta label from Helm documentation {pull}4802[#4802] (issue: {issue}4796[#4796])
+
+[[bug-1.8.0]]
+[float]
+=== Bug fixes
+
+* Fix handling CAs in Agent controller {pull}4807[#4807] (issue: {issue}4790[#4790])
+* Improve Elasticsearch cluster health API call when timeout {pull}4803[#4803] (issue: {issue}4507[#4507])
```
Comment (pebrc) [inline docs/release-notes/1.8.0.asciidoc:29]: Feels like this is only half of a sentence

**Review by pebrc — APPROVED** [review]

File: docs/release-notes/1.8.0.asciidoc (hunk 5 above)
Comment (sebgl) [reply docs/release-notes/1.8.0.asciidoc:29]: 👍 Changing to "Improve Elasticsearch cluster health API timeout handling"

**Pull request merged**